src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -18,8 +30,20 @@
      *                 - DAY - текущий день (число месяца);
      *                 В любом другом случае бросить DateTimeException
      **/
+
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        switch (datePart) {
+            case FULL:
+                return LocalDate.now().toString();
```

Comment:
let's create a variable and save the result of `LocalDate.now()` then we can use this variable at this method